tools/hangar/__snapshots__/error.ts.snap

```diff
@@ -30,7 +30,52 @@ Duration <DURATION>"
 `;
 
 exports[`inflight_stacktraces.test.w 1`] = `
-"fail ┌ inflight_stacktraces.test.wsim » root/env0/test:assert              
+"[ERROR] assert | Error: assertion failed: false
+  --> ../../../examples/tests/error/inflight_stacktraces.test.w:7:3
+  | let bucket = new cloud.Bucket();
+  | 
+  | test \\"assert\\" {
+7 |   assert(false);
+  |   ^
+at <ABSOLUTE>/inflight_stacktraces.test.w:7:3
+[ERROR] expect.equal | Error: Expected values to be strictly deep-equal:
+
+1 !== 2
+
+   --> ../../../examples/tests/error/inflight_stacktraces.test.w:11:3
+   | }
+   | 
+   | test \\"expect.equal\\" {
+11 |   expect.equal(1,2 );
+   |   ^
+at <ABSOLUTE>/inflight_stacktraces.test.w:11:3
+[ERROR] bucket failed delete | Error: Object does not exist (key=doesn't exist).
+[ERROR] bucket failed delete | Error: Object does not exist (key=doesn't exist).
```

Comment:
This is a "caught" exception. By default I don't think we want to see these printed